src/main/java/ua/dymohlo/mongoandjson/Main.java

```diff
@@ -0,0 +1,167 @@
+    }
+
+    private static void jsonPracticeTestReadValue() {
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        JsonPracticeObject newJsonPractice;
+        try {
+            newJsonPractice = objectMapper.readValue(jsonPracticeTestWriteValue(), JsonPracticeObject.class);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+        System.out.println(newJsonPractice);
+
+    }
+
+    private static MongoClient mongoClient;
+
+    private static MongoCollection<Document> connectToMongoDB() {
+        MongoClient mongoClient = MongoClients.create("mongodb://localhost:27017");
+        MongoDatabase database = mongoClient.getDatabase("test");
+        return database.getCollection("humans");
+    }
+
+    private static void closeMongoDBConnection() {
+        if (mongoClient != null) {
+            mongoClient.close();
+            System.out.println("MongoDB connection closed");
+        }
+    }
+
+    private static void mongoPracticCreatePerson() {
+        try {
+            Document doc = new Document("name", "Mr. Robot");
+            connectToMongoDB().insertOne(doc);
+            System.out.println("The Person is created");
+        } finally {
+            closeMongoDBConnection();
+            System.out.println("The operation is over");
+        }
+    }
+
+    private static void mongoPracticFindingByName() {
+        try {
+            FindIterable<Document> documents = connectToMongoDB().find();
+
+            MongoCursor<Document> cursor = documents.iterator();
+
+            while (cursor.hasNext()) {
+                Document next = cursor.next();
+                String fieldValue = next.getString("name");
+
+                if ("Bruce Dickinson".equals(fieldValue)) {
+                    System.out.println(next);
+                }
+            }
+        } finally {
+            closeMongoDBConnection();
+            System.out.println("The operation is over");
+        }
+    }
+
+    private static void mongoPracticFindingById() {
+        try {
+            ObjectId objectId = new ObjectId("658c274ec0ed91694130487e");
+            Document query = new Document("_id", objectId);
+            Document result = connectToMongoDB().find(query).first();
+            if (result != null) {
+                System.out.println("Found document: " + result.toJson());
+            } else {
+                System.out.println("Document not found");
+            }
+        } finally {
+            closeMongoDBConnection();
+            System.out.println("The operation is over");
+        }
+    }
+
+    private static void mongoPracticFindingAll() {
+        try {
+            FindIterable<Document> documents = connectToMongoDB().find();
+            MongoCursor<Document> cursor = documents.cursor();
+            while (cursor.hasNext()) {
+                Document next = cursor.next();
+                System.out.println(next);
+            }
+        } finally {
```

Comment:
Minor note:

Since FindIterable alread is a Iterable object, we just can use for-loop with it, without getting a cursor (because it's a more low level abstraction)